coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -111,6 +126,7 @@ func TestUpdateMetrics_MetricsDoNotExpire(t *testing.T) {
 
 func verifyCollectedMetrics(t *testing.T, expected []*agentproto.Stats_Metric, actual []prometheus.Metric) bool {
 	if len(expected) != len(actual) {
+		t.Logf("expected %d metrics, got %d", len(expected), len(actual))
```

Comment:
nit: won't `t.Logf` be spamming in build logs too much?

Reply:
In local testing I never hit it when the test passed.

Reply:
yeah, I'm thinking about the failure scenario

Reply:
if it's failing, then I want to know, spam be damned

Reply:
Maybe unrelated but Zerolog has an easy "sampler" you can use for this kind of stuff: https://github.com/rs/zerolog?tab=readme-ov-file#log-sampling

Makes it easy to throw into some active loops while still controlling the spam.

Reply:
TIL 👍